packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -1,63 +1,68 @@
 import { waitForRemotePeer } from "@waku/core/lib/wait_for_remote_peer";
-import { createLightNode } from "@waku/create";
-import type { WakuLight } from "@waku/interfaces";
+import { createFullNode } from "@waku/create";
+import type { WakuFull } from "@waku/interfaces";
 import { Protocols } from "@waku/interfaces";
 
-import { makeLogFileName, Nwaku } from "../src";
+import { NOISE_KEY_1, Nwaku } from "../src";
 import { delay } from "../src/delay";
 
 describe("Peer Exchange: Node", () => {
-  let waku: WakuLight;
+  let waku: WakuFull;
   let nwaku1: Nwaku;
   let nwaku2: Nwaku;
   let nwaku3: Nwaku;
-
   afterEach(async function () {
     !!nwaku1 && nwaku1.stop();
     !!nwaku2 && nwaku2.stop();
     !!nwaku3 && nwaku3.stop();
+
     !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
   });
 
-  it("Queries successfully", async function () {
-    this.timeout(15_000);
+  it.only("Queries successfully", async function () {
+    this.timeout(150_000);
 
     console.log("starting");
-    nwaku1 = new Nwaku(makeLogFileName(this));
-    nwaku2 = new Nwaku(makeLogFileName(this));
-    nwaku3 = new Nwaku(makeLogFileName(this));
+    nwaku1 = new Nwaku("node1");
+    nwaku2 = new Nwaku("node2");
+    nwaku3 = new Nwaku("node3");
 
-    await nwaku2.start({ discv5Discovery: true, peerExchange: true });
-    console.log("nwaku2 started");
+    await nwaku1.start({
+      discv5Discovery: true,
+      peerExchange: true,
+      manualArgs: [`--discv5-udp-port:9007`],
+    });
+    console.log("nwaku1 started");
 
-    const enr = (await nwaku2.info()).enrUri;
+    const enr = (await nwaku1.info()).enrUri;
     console.log({ enr });
 
-    await nwaku1.start({
+    await nwaku2.start({
       discv5Discovery: true,
-      manualArgs: [
-        `--discv5-bootstrap-node:${(await nwaku2.info()).enrUri}`,
-        `--ports-shift=5`,
-      ],
+      manualArgs: [`--discv5-bootstrap-node:${enr}`, `--discv5-udp-port:9043`],
```

Comment:
Please remove the `manualArgs` and add `discv5BootstrapNode` to the args and include `discv5-udp-port` to the port finder logic in nwaku @danisharora099